cinema/views.py

```diff
@@ -1 +1,69 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+from cinema.serializers import (
+    ActorSerializer,
+    GenreSerializer,
+    CinemaHallSerializer,
+    MovieSerializer,
+    MovieSessionSerializer,
+    MovieSessionListSerializer,
+    MovieListSerializer,
+    MovieDetailSerializer,
+    MovieSessionDetailSerializer,
+)
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        if self.action not in ["list", "retrieve"]:
```

Comment:
tuples are faster than lists. Since you don't need to somehow modify this data structure - tuple is preferable here